examples/todomvc/src/components/footer/footer.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { onlyUpdateForPropTypes } from 'recompose'
+import classnames from 'classnames'
+
+const Footer = ({
+  // style
+  style,
+  className,
+  // data
+  todos,
+  todosLeft,
+  todosCompleted,
+  filter,
+  // callbacks
+  onClearCompleted,
+  onClickAll,
+  onClickCompleted,
+  onClickActive,
+}) => {
+  if (todos === 0) return null
+  return (
+    <footer style={style} className={`footer ${className}`}>
+      <span className="todo-count">
+        <strong>{todosLeft}</strong> {`item${todosLeft > 1 ? 's' : ''} left`}
+      </span>
+      <ul className="filters">
+        <li>
+          <a className={classnames({ selected: filter === 'all' })} onClick={onClickAll}>All</a>
+        </li>
+        <li>
+          <a className={classnames({ selected: filter === 'active' })} onClick={onClickActive}>Active</a>
+        </li>
+        <li>
+          <a className={classnames({ selected: filter === 'completed' })} onClick={onClickCompleted}>Completed</a>
```

Comment:
To avoid long lines like that, I'm used to import `classnames` lib with name `cn` : 
``` jsx
import cn from 'classnames'
...
<a className={cn({ selected: filter === 'all' })} onClick={onClickAll}>All</a>
```